Description: Ignore failure of unit test base_utility/comparepasswords_issafe
 as volatile system load may cause false negatives
Author: "Alexander A. Klimov" <alexander.klimov@icinga.com>
Origin: https://github.com/Icinga/icinga2/pull/6977/commits/646feb76e06155f908a4dcee51144db8f05a114f
Bug: https://github.com/Icinga/icinga2/issues/6987

--- a/test/base-utility.cpp
+++ b/test/base-utility.cpp
@@ -64,7 +64,7 @@ BOOST_AUTO_TEST_CASE(comparepasswords_is
 	auto duration2 (steady_clock::now() - start2);
 
 	double diff = (double)duration_cast<microseconds>(duration1).count() / (double)duration_cast<microseconds>(duration2).count();
-	BOOST_CHECK(0.9 <= diff && diff <= 1.1);
+	BOOST_WARN(0.9 <= diff && diff <= 1.1);
 }
 
 BOOST_AUTO_TEST_SUITE_END()
